### DISCUSSION TITLE
When creating an excel file, how do I write to a specific cell?

### DISCUSSION THREAD

**Discussion by weirdyang** [mini-software/MiniExcel#685] [Q&A]
When creating an excel file, how do I write to a specific cell?

For example, I want to open an excel file and write a value to specific cell (e.g. B3) and save it.
Is this possible?

**Comment by michelebastione** [comment]
This is currently not possible, but as a workaround you could look into [MiniExcel templating](https://learn.microsoft.com/dotnet/fundamentals/code-analysis/quality-rules/ca2022), it might fit your use case.

**Comment by weirdyang** [comment]
What I ended up doing is reading it into a list of dynamic then updating the object at the index corresponding to my target row and updating the property corresponding to my target column  and then exporting it again
